packages/contracts/test/registry/dao-registry.ts

```diff
@@ -75,10 +75,10 @@ describe('DAORegistry', function () {
     );
   });
 
-  it('reverts the registration if the DAO name is empty', async function () {
+  it('succeeds even if the dao subdomain is empty', async function () {
     await expect(
       daoRegistry.register(targetDao.address, ownerAddress, '')
-    ).to.be.revertedWithCustomError(daoRegistry, 'EmptyDaoName');
+    ).to.not.be.reverted;
   });
 
```

Comment:
But wait, how could this old code still be in place?

We have just merged the new ENS registration logic, and the test was explicitly updated to handle this case?